newgrf/474e474e/global.yaml

```diff
@@ -1,9 +1,3 @@
 name: "Light Road Vehicle Set"
-replaced-by:
-  unique-id: "4e474c56"
```

Comment:
This is really bad, and shouldn't happen. Need to validate why this sneaked in, and if more of these issues exist.